### DISCUSSION TITLE
[Bug]:  Config Editor catch-22

### DISCUSSION THREAD

**Discussion by nottledim** [blakeblackshear/frigate#14640] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

I'm editing the config file with the built in editor, when I save I get an error.  The error is about 30 lines of red writing which fills the top half of my screen.  The bottom half shows the top of my config file.  I can't scroll down to the bottom of my config file where my error is.  I can't save the file.  

### Steps to reproduce

1. Open config file editor.  My file was 55 lines long.
2. Add something near the bottom which causes a long trace back error.  For example an indentation error in the last block
3. Click "Save Only".
4. Admire the traceback :)
5. Scroll down to bottom of the file - if you can - and fix it. Oh, you can't.
6. Tear your hair out :-)
...

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
any will do
```

### docker-compose file or Docker CLI command

```yaml
n/a
```

### Relevant Frigate log output

```shell
n/a
```

### Relevant go2rtc log output

```shell
n/a
```

### Operating system

Other Linux

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

Neos Smart Cam

### Screenshots of the Frigate UI's System metrics pages

![PXL_20241028_154914011](https://github.com/user-attachments/assets/23f1edf1-af63-4c52-91cb-f463f14df1b5)

### Any other information that may be helpful

Similar problem with Firefox

**Comment by hawkeye217** [comment, marked as answer]
This has already been fixed for the next version of Frigate.

https://github.com/blakeblackshear/frigate/pull/14264
